algorithmia/metadata.csv

```diff
@@ -0,0 +1,2 @@
+metric_name,metric_type,interval,unit_name,per_unit_name,description,orientation,integration,short_name
+duration_milliseconds,gauge,,millisecond,,Duration of algorithm run,0,algorithmia,
```

Comment:
I think we're still missing a metric name here. Is it just the 1 metric?

Reply:
Good question! I can give you some context, and perhaps you could give me some guidance as to whether I should list some other values here.

A example of a payload from Algorithmia to the Datadog metrics API looks like:

```
{
  "time": "2020-09-29T05:02:13.546283",
  "duration_milliseconds": 16,
  "session_id": "rses-981e1fbf-e3a6-44ae-ad33-cc00599bc687",
  "request_id": "req-d717d20b-49e9-466e-a370-1a6cf405486b",
  "algorithm_owner": "koverholt",
  "algorithm_name": "model",
  "algorithm_version": "1.0.0",
  "<USER-DEFINED-METRIC-1>": 4,
  "<USER-DEFINED-METRIC-2>": 0.98,
  "<USER-DEFINED-METRIC-3>": 1,
  "<USER-DEFINED-METRIC-N>": 0.57
}
```

where `<USER-DEFINED-METRIC-N>` are defined by the user in our platform but are not present by default, `duration_milliseconds` is currently listed as the only metric in `metadata.csv`, and all of the other key/values would be used as facets in Datadog rather than metrics.

So, given that information, should I include the keys/values in `metadata.csv` that would be considered to be facets in Datadog? If so, then I can add them. If not, then the only pre-defined metric from a default payload would be `duration_milliseconds` as currently shown in `metadata.csv`.

Reply:
metatdata.csv should only include metrics with stable/known names with consistent units.